Packs/rasterize/ReleaseNotes/2_0_26.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Added the parameter **List of domains to block** to allow an adjustment to block requests from specific domains.
```

Comment:
Added the **List of domains to block** parameter, which allows an adjustment to block requests from specific domains.